docs/sources/reference/api/docker_remote_api_v1.18.md

```diff
@@ -1042,6 +1042,38 @@ Status Codes:
           }
         ]
 
+** Example request, with digest information**:
+
+        GET /images/json?digests=1 HTTP/1.1
+
+**Example response, with digest information**:
+
+        HTTP/1.1 200 OK
+        Content-Type: application/json
+
+        [
+          {
+            "Created": 1420064636,
+            "Id": "4986bf8c15363d1c5d15512d5266f8777bfba4974ac56e3270e7760f6f0a8125",
+            "ParentId": "ea13149945cb6b1e746bf28032f02e9b5a793523481a0a18645fc77ad53c4ea2",
+            "RepoDigests": [
+              "localhost:5000/test/busybox@sha256:cbbf2f9a99b47fc460d422812b6a5adff7dfee951d8fa2e4a98caa0382cfbdbf"
+            ],
+            "RepoTags": [
+              "localhost:5000/test/busybox:latest",
+              "busybox:latest",
+            ],
+            "Size": 0,
+            "VirtualSize": 2429728
+          }
+        ]
+
+In the example above, this image ID is associated with 1 digest for the
```

Comment:
The response shows a single image `Id` associated with the `busybox` repository. The repository appears in the `RepoDigests` array: `localhost:5000/test/busybox@sha...`  A caller can use the `busybox:latest` tag in the `RepoTags` array to reference the image.

The association is betwen the image and the digest.  The `busybox` repository has no association to the digest. This means a caller cannot reference the image using the `RepoDigests` value ( `localhost:5000/test/busybox@sha...` ).